core/src/test/kotlin/ch/ergon/dope/QueryBuilderTest.kt

```diff
@@ -491,4 +494,30 @@ class QueryBuilderTest {
 
         assertEquals(expected, actual)
     }
+
+    @Test
+    fun `should support selecting all Fields from bucket`() {
+        val someBucket = UnaliasedBucket("someBucket")
```

Comment:
Can you use _someBucket()_ you added in the other PR?

Reply:
done